test/server/services/user.js

```diff
@@ -109,6 +109,19 @@ describe('User service tests', () => {
           done();
         }, done);
     });
+    it('should retrieve all users', (done) => {
+      User.getAllUsers()
+        .then((users) => {
+          const { models } = users;
+          const [entry1, entry2] = models;
```

Comment:
That definitely could work. I was being more conservative since I didn't want to inadvertently limit the models array to only 2 elements, just in case there's more.